trace/tr_test.go

```diff
@@ -0,0 +1,150 @@
+package trace_test
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/TyeMcQueen/go-lager"
+	"github.com/TyeMcQueen/go-lager/gcp-spans"
+	"github.com/TyeMcQueen/go-tutl"
+	"github.com/TyeMcQueen/tools-gcp/trace"
+)
+
+func TestTrace(t *testing.T) {
+	u := tutl.New(t)
+	logs := &bytes.Buffer{}
+	defer u.Is("", logs.Bytes(), "no final errors")
+	defer lager.SetOutput(logs)()
+
+	proj := os.Getenv("GCP_PROJECT_ID")
+	if "" == os.Getenv("GCP_PROJECT_ID") {
+		out, err := os.OpenFile("/dev/tty", os.O_WRONLY, 0666)
+		if nil != err {
+			u.Log("Can't write to /dev/tty: ", err)
+			out = os.Stderr
+		}
+		fmt.Fprintf(out, "SKIPPED: %s.\n",
+			"Testing requires GCP_PROJECT_ID set and default creds")
+		t.SkipNow()
```

Comment:
should we error instead of skipping?

Reply:
No, it can be a bit of a pain to set up default credentials, so I don't think that should be required to have tests pass.  The test does go out of its way to not skip silently, though.